platforms/Windows/runtime-amd64.wxs

```diff
@@ -13,8 +13,12 @@
     <!-- Directory Structure -->
     <StandardDirectory Id="ProgramFiles64Folder">
       <Directory Id="INSTALLDIR" Name="Swift">
-        <!-- TODO(compnerd) use $(var.ProductVersion) -->
-        <Directory Id="_" Name="runtime-development">
+        <?if $(var.ProductVersion) = "0.0.0"?>
```

Comment:
If this isn't scoping dependent and is referenced later in the file, shouldn't we pull it to the top level and possibly at the top of the file? (same for other files)